sources/Interop/Windows/um/dxcapi/IDxcBlobEncoding.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from DirectXShaderCompiler, include/dxc/dxcapi.h, at commit d356a8bc546da38612cac13d1e643de17a0ba38f
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct IDxcBlobEncoding
+    {
+        public Vtbl* lpVtbl;
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        [return: NativeTypeName("HRESULT")]
+        public delegate int _QueryInterface(IDxcBlobEncoding* pThis, [NativeTypeName("const IID &")] Guid* riid, [NativeTypeName("void **")] void** ppvObject);
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        [return: NativeTypeName("ULONG")]
+        public delegate uint _AddRef(IDxcBlobEncoding* pThis);
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        [return: NativeTypeName("ULONG")]
+        public delegate uint _Release(IDxcBlobEncoding* pThis);
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        [return: NativeTypeName("LPVOID")]
+        public delegate void* _GetBufferPointer(IDxcBlobEncoding* pThis);
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        [return: NativeTypeName("SIZE_T")]
+        public delegate uint _GetBufferSize(IDxcBlobEncoding* pThis);
```

Comment:
The shared remap will help avoid issues like this and will cause `nuint` to be emitted rather than `uint`